package.json

```diff
@@ -86,9 +86,7 @@
   },
   "packageManager": "yarn@4.1.0",
   "lint-staged": {
-    "*": [
-      "eslint --fix --report-unused-disable-directives",
-      "prettier --write"
-    ]
+    "*.{js,mjs,cjs,ts,mts,cts}": "eslint --fix --report-unused-disable-directives",
+    "*": "prettier --write --ignore-unknown"
```

Comment:
These will run in parallel now iirc, we should disable the concurrency so eslint is done first and prettier after

Reply:
Done